gatsbyjs/package.json

```diff
@@ -8,25 +8,28 @@
     "gatsby"
   ],
   "scripts": {
-    "develop": "gatsby develop",
+    "develop": "ENABLE_GATSBY_REFRESH_ENDPOINT=true gatsby develop",
     "start": "gatsby develop",
     "build": "gatsby build",
     "clean": "gatsby clean"
   },
   "license": "0BSD",
   "dependencies": {
+    "@storyblok/js": "^1.0.4",
+    "@storyblok/react": "^1.0.4",
     "@storyblok/storyblok-editable": "^1.0.0",
```

Comment:
We should remove this package since it's not used anymore, right?

Reply:
@fgiuliani We still need both of them as I mentioned the workaround in the comment above.
Until gatsby-source-storyblok will be updated, we need both to be included as current gatsby-source-storyblok doesn't provide a few APIs that we built in JS & React SDKs.

Reply:
But where is the package `@storyblok/storyblok-editable` being used? Can't we use the `storyblokEditable` function provided by `@storyblok/react` instead, which has the same purpose?

Reply:
Ah! I misunderstood you were talking about JS & React SDK. I totally missed storyblok-editable because it was not highlighted. Removed together with js SDK and I'll push with other fixes.